src/components/WebexActivityStream/WebexActivityStream.js

```diff
@@ -138,19 +170,33 @@ Greeting.propTypes = {
   personName: PropTypes.string.isRequired,
 };
 
+export function TimeRuler({text}) {
```

Comment:
any reason we have the extra `{}` in the params here?

Reply:
Destructures the props object so we don't have to do `props.text`.

Reply:
I wish we could have done that for all other components. now there is inconsistency on how we extract the props.

Reply:
I can change to `props` if needed for consistency.

Reply:
no, I actually like it better. Lalli suggested to fix it for other components too. it will save us some lines total.

Reply:
I'm already using it, and will fix it on my upcoming PR